doc/client_api.md

```diff
@@ -391,9 +391,19 @@ room.publish(localStream, {maxVideoBW:300}, function(id, error){
   } else {
     console.log("Published stream", id);
   }
-});            
+});
 ```
 
+### Publish Simulcast Streams
+
+You can enable Simulcast in the publisher by adding the next option:
```

Comment:
I'd warn here of the current simulcast limitation: i.e. Chrome only and no recording.